aiinfra-cluster/variables.tf

```diff
@@ -72,6 +73,7 @@ variable "instance_count" {
 variable "accelerator_type" {
   description = "The accelerator (GPU) type."
   type        = string
+  default     = "nvidia-tesla-a100"
```

Comment:
does this work for migs which I thought from [this](https://cloud.google.com/compute/docs/gpus#a100-gpus) requires the machine type to be an a2 if the desired gpu is an a100?

Reply:
default GPU count is set to 0. So it works as is. But if the user changes the GPU count, they have to choose the right vm type. Hopefully we will get to determine the VM type ourselves based on number of GPUs asked.